### DISCUSSION TITLE
Pkl over Cue

### DISCUSSION THREAD

**Discussion by mbaykara** [apple/pkl#76] [General]
What are the advantages of `Pkl` over `Cue`? What arguments speak in favor of using or switching from `Cue` to `Pkl`?

**Comment by GeoffreyStemys** [comment]
More output language support with pkl

**Comment by gordonbondon** [comment]
Some of these seems to be answered in https://github.com/apple/pkl/discussions/7

**Comment by holzensp** [comment]
Marking this as duplicate of #7 to have all discussion on the topic in one place.
